src/Components/ToDoList/ToDoList.tsx

```diff
@@ -1,15 +1,37 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
+
 
 import ToDoItem from '../ToDoItem/ToDoItem';
 
 const ToDoList = () => {
-    return(
+
+    const listItems = [
+        {
+            id:0,
+            title: 'Task 1',
+            desc: "Lorem ipsum set dolores"
+        },
+        {
+            id:1,
+            title: 'Task 2',
+            desc: "Lorem ipsum set dolores uno dos tres"
+        },
+        {
+            id:2,
+            title: 'Task 3',
+            desc: "Hello World"
+        }
+    ]
+
+    const toDoListItems = (): ReactNode => {
+        return (
+        listItems.map( item => <ToDoItem key={item.id} {...item}/> )
+        ) 

```

Comment:
could be shorter:
const renderTodos = (): ReactNode => listItems.map( item => <ToDoItem key={item.id} {...item}/>